bigquery_etl/cli/query.py

```diff
@@ -117,7 +118,41 @@ def query(ctx):
     default=False,
     is_flag=True,
 )
-def create(name, sql_dir, project_id, owner, init):
+@click.option(
+    "--dag",
+    "-dag",
+    help=(
+        "Name of the DAG the query should be scheduled under. "
+        "To see available DAGs run `bqetl dag info`. "
+        "To create a new DAG run `bqetl dag create`. To skip the "
+        " automated scheduling use option --no_schedule."
+    ),
+)
+@click.option(
+    "--use_default_dag",
+    "-default-dag",
```

Comment:
Our convention for longer options is to allow underscores or dashes as separators between the words in the option, but they should always start with two dashes and otherwise match.
```suggestion
    "--use_default_dag",
    "--use-default-dag",
```
However, I agree with @ANich's suggestion to have `bqetl_default` be the default value for the `--dag` option instead of having this option.

Reply:
I would +1 to using --use_default_dag
We talked about this during last Infraday. In summary, even though sending a query to the `bqetl_default` makes it easier for the developer, we'd like to ensure awareness when scheduling a query there and avoid a long list of queries in this default_dag with not much clarity about the business area or project they belong to.

Pinging the people involved in the Infraday chat about your preference @curtis @kik-kik @akkomar @lelilia @scholtzan?

Reply:
I agree with both of you - `--use_default_dag` seems redundant and at the same time it would be unfortunate if `bqetl_default` became a mess.
One way for ensuring developer awareness I can think of is to add some comment line above a task added to `bqetl_default`, like ```# Automatically scheduled by `bqetl query create` called without `--dag` option.```. This way they'd notice this at the latest in a PR diff.

Reply:
You could also add a check to `query_scheduling.utils.is_valid_dag_name` to mark  `bqetl_default` as invalid

Reply:
+1 to akkomar's suggestion. I guess it is also partly the reviewers responsibility to make sure `bqetl_default` doesn't become to messy

Reply:
How would such a comment be added though?  The `metadata.yaml` files are generated from Python data structures which don't seem to allow for inserting comments in the generated YAML, and in my experience any time the `bqetl` tooling messes with `metadata.yaml` files it strips any existing comments from them.

Reply:
We could add a label or a key under `scheduling` in `metadata.yaml` like `default_scheduling_warning: true` and use it in [DAG template](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/templates/airflow_dag.j2) to decide on adding a comment there.

Reply:
A comment in the DAG would be OK, but that's not where people are configuring the value so I'd think that'd be much less likely to get noticed/heeded (especially if we ever get the DAG generation completely automated so we don't have to manually generate and check in the DAGs, which I think is still the plan).

If you do go that route, it seems like it could just check whether the `dag` value is "bqetl_default" (no separate property necessary).

Reply:
I set a default dag name instead of using the option --use-default and added a WARNING to the DAG template and to the output of the query create command. Should be visible both for author and reviewer.